packages/training/deploy/DEPLOY_TESTING.md

```
@@ -0,0 +1,1115 @@
+python deploy/runpod/setup.py list
+
+# Check logs (opens web console)
+python deploy/runpod/setup.py logs <pod-id>
+
+# SSH into pod (from RunPod console)
+# Then check same things as container debugging above
+```
+
+### Specific Error Solutions
+
+**"Empty batch, skipping step" on every step**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)